.github/workflows/local-test.yml

```diff
@@ -2,9 +2,9 @@ name: Integration Test
 
 on:
   push:
-    branches: [ dev ]
+    branches: [ dev, test-ci ]
```

Comment:
I added this because it's convenient to have a dedicated, unprotected branch were we can run the CI steps.